### PR TITLE
feat: add more info, renderer => viewer, info box on top

### PR DESCRIPTION
**Pull request by mcmcgrath13**
* Mention OSOM and add link to the information section
* Use config to render control buttons
* Add info box on what people are looking at as quick intro
* Use `Narragansett Bay Volume Viewer` and similar language everywhere

### CODE REVIEW

**Review by kmilo9999 — APPROVED** [review]
looks good to me!

File: src/components/React/ControlsPanel.js [hunk 1]
```diff
@@ -45,12 +45,17 @@ export default function ControlPanel(props) {
               });
               // Change color map
               reduxDispatch(
-                changeColorMap(val === "salt" ? colorMaps[0] : colorMaps[1])
+                changeColorMap(
+                  val.value === "salt"
+                    ? colorMaps.find((m) => m.name === "Haline")
+                    : colorMaps.find((m) => m.name === "Thermal")
```
Comment (RobertGemmaJr) [inline src/components/React/ControlsPanel.js:51]: I knew there had to be a way to do this without a hard coded index! I'll have to take a quick look at the method

**Review by RobertGemmaJr — CHANGES_REQUESTED** [review]
Looks great! Compile warnings prevented firebase from running though.

Otherwise 🚀

File: src/assets/config.json [hunk 2]
```diff
@@ -1,17 +1,61 @@
...
+      "name": "Temperature"
+    }
+  ],
   "slices": 55,
-  "x_spacing": 2.0,
-  "y_spacing": 2.0,
-  "z_spacing": 1.0,
+  "x_spacing": 2,
+  "y_spacing": 2,
+  "z_spacing": 1,
```
[the remark this answers is not in the document]
Comment (mcmcgrath13) [reply src/assets/config.json:37]: in javascript everything is a `Number`, which is secretly a float, so I don't think this matters - my prettifier did this for me and it seems to load fine?

**Review by RobertGemmaJr — APPROVED** [review]

**Pull request merged**